pkgs/applications/misc/tootstream/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname   = "tootstream";
+  version = "0.3.7";
+
+  src = fetchFromGitHub {
+    owner  = "magicalraccoon";
+    repo   = "tootstream";
+    rev    = "${version}";
+    sha256 = "00ajry7ac9a8x661bkk1hgp6afaqxc1x34w0wbad40bprdvdzhl3";
+  };
+
+  propagatedBuildInputs = with python3Packages;
+    [click mastodonpy colored humanize emoji];
```

I believe this is the preferred multiline format
```suggestion
  propagatedBuildInputs = with python3Packages; [
    click
    mastodonpy
    colored
    humanize
    emoji
  ];
```